fenix/app/src/androidTest/java/org/mozilla/fenix/experimentintegration/generate_smoke_tests.py

```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+import subprocess
+
+import yaml
+
+
+def search_for_smoke_tests(tests_name):
+    """Searches for smoke tests within the requested test module."""
+    path = Path("../ui")
+    files = sorted([x for x in path.iterdir() if x.is_file()])
+    locations = []
+    file_name = None
+    test_names = []
+
+    for name in files:
+        if tests_name in name.name:
+            file_name = name
+            break
+
+    with open(file_name, "r") as file:
+        code = file.read().split(" ")
+        code = [item for item in code if item != ""]
+
+        for count, item in enumerate(code):
+            if "class" in item:
+                locations.append(count)
+            if "@SmokeTest" in item:
+                locations.append(count)
```

Comment:
Just checking... is it intended that `count` would be appended twice should `"class"` and `"@SmokeTest"` be found in item?

At first glance I thought this, but maybe not...
```
if "class" in item or "@SmokeTest" in item:
    locations.append(count)
```